src/engine/level_script.c

```diff
@@ -942,10 +951,23 @@ struct LevelCommand *level_script_execute(struct LevelCommand *cmd) {
     sScriptStatus = SCRIPT_RUNNING;
     sCurrentCmd = cmd;
 
+#ifdef PUPPYPRINT_DEBUG
+    u32 first = osGetCount();
+#endif
+
     while (sScriptStatus == SCRIPT_RUNNING) {
         LevelScriptJumpTable[sCurrentCmd->type]();
     }
 
+#ifdef PUPPYPRINT_DEBUG
+    if (gLoadLevel && gLoadLevelAreaTime) {
+        u32 totalTime = gLoadLevelAreaTime + (osGetCount() - first);
```

Comment:
Isn't gLoadLevelAreaTime unnecessary? I feel like it's getting counted more than once.

Reply:
I did not write this code, @FazanaJ needs to elaborate. From what I see it is needed though.

Reply:
This is necessary

Reply:
No, looking back at it this is definitely wrong. `gLoadLevelAreaTime` can be used as a factor to the if statement I guess, but just the `osGetCount() - first` is all that's needed here. `init_level` is getting invoked after `first` has already been set.

Reply:
Ultimately this was the product of a 5 minute, if that, attempt to get a better readout. I never checked that it was accurate, just representative. Ideally something better would be done to give an accurate readout but that was effort I wasn't gonna do at the time

Reply:
Actually on second glance, this is still extremely inaccurate because it also throws an entire `level_update` call into the reading.

Reply:
Either the profiling on this should be relatively accurate or this needs to be purged from the repo entirely, since we're better off with no profiling than bad profiling.